client/grpc/tmservice/service.go

```diff
@@ -88,6 +87,13 @@ func (s queryServer) GetBlockByHeight(ctx context.Context, req *GetBlockByHeight
 
 // GetBlockByHash implements ServiceServer.GetBlockByHash
 func (s queryServer) GetBlockByHash(_ context.Context, req *GetBlockByHashRequest) (*GetBlockByHashResponse, error) {
+	if n := len(req.Hash); n != 32 {
```

Comment:
sorry but what is "nit" meaning?

Reply:
Oh! How about using defined value instead of `32`?

Reply:
Sorry for my short message. How about using the defined `Size` or not?

https://google.github.io/eng-practices/review/reviewer/standard.html

> Reviewers should always feel free to leave comments expressing that something could be better, but if it’s not very important, prefix it with something like “Nit: “ to let the author know that it’s just a point of polish that they could choose to ignore.

> Code review can have an important function of teaching developers something new about a language, a framework, or general software design principles. It’s always fine to leave comments that help a developer learn something new. Sharing knowledge is part of improving the code health of a system over time. Just keep in mind that if your comment is purely educational, but not critical to meeting the standards described in this document, prefix it with “Nit: “ or otherwise indicate that it’s not mandatory for the author to resolve it in this CL.